pkg/generate/app/pipeline.go

```diff
@@ -91,7 +91,7 @@ func (p *Pipeline) NeedsDeployment(env Environment, name string) error {
 // Objects converts all the components in the pipeline into runtime objects
 func (p *Pipeline) Objects(accept, objectAccept Acceptor) (Objects, error) {
 	objects := Objects{}
-	if p.InputImage != nil && p.InputImage.AsImageStream && accept.Accept(p.InputImage) {
+	if p.InputImage != nil && accept.Accept(p.InputImage) {
```

Comment:
Can you explain why this flag is not needed anymore?  If it's not needed, why did the flags above change?